install.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+
+echo Installing protostar
+
+PROTOSTAR_DIR=${PROTOSTAR_DIR-"$HOME/.protostar"}
+PROTOSTAR_BIN_DIR="$PROTOSTAR_DIR/bin"
+mkdir -p $PROTOSTAR_BIN_DIR
+
+PROTOSTAR_REPO="https://github.com/arcticae/testrepo" # TODO: Pull binary from repo
```

Comment:
.

Reply:
This script is not ready, will be edited in next PR